src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -351,6 +356,47 @@ private void markDirty()
         isDirty = true;
     }
 
+    /**
+     * Increment the mobs killed by this colony.
+     * <p>
+     * Will award achievements for mobs killed.
+     */
+    public void incrementMobsKilled()
+    {
+        killedMobs++;
+        final int mobKills = this.getKilledMobs();
+        if (mobKills >= 1)
+        {
+            this.triggerAchievement(ModAchievements.achievementKillOneMob);
+        }
+        if (mobKills >= 25)
+        {
+            this.triggerAchievement(ModAchievements.achievementKill25Mobs);
+        }
+        if (mobKills >= 100)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)